packages/esm-patient-chart-app/src/config-schema.ts

```diff
@@ -118,6 +118,11 @@ export const esmPatientChartSchema = {
       "Whether the visit location field in the Start Visit form should be view-only. If so, the visit location will always be set to the user's login location.",
     _default: false,
   },
+  showBillingSlot: {
+    _type: Type.Boolean,
+    _description: 'Whether on start visit form to show billing extension and enable billing functionality',
```

Comment:
Description seems ambiguous.

suggestion: "Whether to show the billing extension and enable billing functionality, on the start visit form."